packages/fela-dom/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fela-dom",
-  "version": "7.0.7",
+  "version": "7.0.8",
```

Comment:
No need to manually bump the version, the release process will take care of it

Reply:
Removed, thanks for notify.